### PR TITLE
RSA8f2

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Spec/Auth.swift [hunk 1]
```diff
@@ -713,6 +713,27 @@ class Auth : QuickSpec {
                 }
 
             }
+
+            // RSA8f2
+            it("ensure that the message is rejected") {
+                let token = getTestToken()
+                let options = ARTClientOptions(token: token)
```
Comment (mattheworiordan) [inline Spec/Auth.swift]: Has this token got a null value clientId?

File: Spec/Auth.swift [hunk 2]
```diff
@@ -713,6 +713,27 @@ class Auth : QuickSpec {
...
+                options.environment = "sandbox"
+                options.clientId = "john"
```
Comment (mattheworiordan) [inline Spec/Auth.swift]: Like RSA8f3 & RSA8f4, this is wrong.

File: Spec/Auth.swift (hunk 1 above)
Comment (ricardopereira) [inline Spec/Auth.swift]: Done 70ab463.

File: Spec/Auth.swift (hunk 2 above)
Comment (ricardopereira) [inline Spec/Auth.swift]: Done 70ab463.

**Comment by tcard** [comment]
LGTM

**Pull request merged**